data/json/vehicles/cars.json

```
@@ -713,6 +713,81 @@
     ],
     "items": [ { "x": -1, "y": 0, "chance": 8, "items": [ "sm_extinguisher" ] } ]
   },
+  {
+    "id": "car_racing_electric",
+    "type": "vehicle",
+    "name": "Electric Racing Car",
+    "blueprint": [
+      [ "o-+-o" ],
+      [ "+=#'|" ],
+      [ "+=#'|" ],
+      [ "o-+-o" ]
+    ],
+    "parts": [
+      { "x": 0, "y": 0, "part": "frame_vertical_2" },
+      { "x": 0, "y": 0, "part": "seat" },
+      { "x": 0, "y": 0, "part": "seatbelt" },
+      { "x": 0, "y": 0, "part": "controls" },
+      { "x": 0, "y": 0, "part": "dashboard" },
+      { "x": 0, "y": 0, "part": "vehicle_alarm" },
+      { "x": 0, "y": 0, "part": "horn_car" },
+      { "x": 0, "y": 0, "part": "roof" },
```

```suggestion
      { "x": 0, "y": 0, "parts": [ "xlframe_vertical_2", "seat", "seatbelt", "controls", "dashboard", "vehicle_alarm", "horn_car", "roof" ] },
```
Please use the parts array for a more compact listing.